src/fidesctl/api/ctl/sql_models.py

```diff
@@ -240,3 +240,5 @@ class System(Base, FidesBase):
     "system": System,
     "evaluation": Evaluation,
 }
+
+taxonomy_types = [DataCategory, DataQualifier, DataSubject, DataUse]
```

Comment:
i would make this a little bit smarter, maybe iterate through the models and check which ones have an `is_default` field?
otherwise I guarantee I'll go to update this code in a few months and get stuck for 3 hours 😂